js/pages/footer.js

```diff
@@ -1,38 +1,38 @@
                     <div class="dceg-footer align-left text-center text-sm-left">Division of Cancer Epidemiology and Genetics</div>
                     <div class="nci-dceg-footer align-left text-center text-sm-left">at the National Cancer Institute</div>
                 </div>
             </div>
             <div class="row w-100 pt-0 align-left" style="margin:0">
                 <div class="col-lg-4 pt-3 content-wrapper text-center text-sm-left">
-                    <div class="footer-heading">HAVE QUESTIONS?</div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://norcfedramp.servicenowservices.com/recruit">Contact the Connect Support Center</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" href="mailto:ConnectStudy@norc.org">Email Us</a></div>
+                    <div class="footer-heading" data-i18n="footer.Questions">HAVE QUESTIONS?</div>
+                    <div class="footer-sub-heading pt-3" data-i18n="footer.contactLink"><a class="footer-links" target="__blank" href="https://norcfedramp.servicenowservices.com/recruit">Contact the Connect Support Center</a></div>
+                    <div class="footer-sub-heading pt-3" data-i18n="footer.emailUs"><a class="footer-links" href="mailto:ConnectStudy@norc.org">Email Us</a></div>
                 </div>
                 <div class="col-lg-4 pt-3 content-wrapper text-center text-sm-left">
-                    <div class="footer-heading">POLICIES</div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://www.cancer.gov/global/web/policies/accessibility">Accessibility</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://www.cancer.gov/policies/disclaimer">Disclaimer</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://www.cancer.gov/global/web/policies/foia">FOIA</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://www.cancer.gov/policies/privacy-security">Privacy & Security</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html">HHS Vulnerability Disclosure</a></div>
-                    <div class="footer-sub-heading pt-3"><a class="footer-links" target="__blank" href="https://dceg.cancer.gov/about/sitemap">Site Map</a></div>
+                    <div class="footer-heading" data-i18n="footer.policies">POLICIES</div>
+                    <div class="footer-sub-heading pt-3" data-i18n="footer.accessibility"><a class="footer-links" target="__blank" href="https://www.cancer.gov/global/web/policies/accessibility">Accessibilit</a></div>
```

Comment:
Should this be `Accessibility`?

Reply:
It is in the translation files but I will update it